tests/dag_processing/test_processor.py

```diff
@@ -160,6 +160,35 @@ def dag_in_a_fn():
         assert result.import_errors == {}
         assert result.serialized_dags[0].dag_id == "test_abc"
 
+    def test_top_level_connection_access(self, tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch):
+        # Create the dag in a fn, and use inspect.getsource to write it to a file so that
+        # a) the test dag is directly viewable here in the tests
+        # b) that it shows to IDEs/mypy etc.
```

Comment:
We don't need this comment block each time - perhaps move it to the writer function?

Reply:
Let me try and do that.

Reply:
Handled in latest commit